ariadne_codegen/client_generators/client.py

```diff
@@ -205,9 +229,11 @@ def _generate_subscription_method_def(
                 value=generate_name(ASYNC_ITERATOR), slice_=generate_name(return_type)
             ),
             body=[
-                self._generate_operation_str_assign(operation_str, 1),
-                self._generate_variables_assign(arguments_dict, 2),
-                self._generate_async_generator_loop(operation_name, return_type, 3),
+                self._generate_operation_str_assign(variable_names, operation_str, 1),
```

Comment:
We now have inconsistency where some functions get `variable_names` as first arg, and some as last arg. I guess we should move it to last arg everywhere because that makes more sense in other places.

Reply:
only 3 functions have `variable_name` as last argument:
https://github.com/mirumee/ariadne-codegen/blob/6438dfb7dc0fe117ee69a444874b6dc66569a627/ariadne_codegen/client_generators/client.py#L148
https://github.com/mirumee/ariadne-codegen/blob/6438dfb7dc0fe117ee69a444874b6dc66569a627/ariadne_codegen/client_generators/client.py#L159
https://github.com/mirumee/ariadne-codegen/blob/6438dfb7dc0fe117ee69a444874b6dc66569a627/ariadne_codegen/client_generators/client.py#L169
Would it not be better to change there to first argument? 
Some [functions](https://github.com/mirumee/ariadne-codegen/blob/6438dfb7dc0fe117ee69a444874b6dc66569a627/ariadne_codegen/client_generators/client.py#L300-L302) have default values in the arguments,  which makes it difficult to add arguments as last.

Reply:
Lets leave it as it is then.